sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -905,55 +929,63 @@ export type SignatureAlgorithm =
   | "ES256K";
 
 /**
- * Options for the encrypt call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link encrypt}.
  */
 export interface EncryptOptions extends CryptographyOptions {}
 
 /**
- * Options for the decrypt call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link decrypt}.
  */
 export interface DecryptOptions extends CryptographyOptions {}
 
 /**
- * Options for the sign call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link sign}.
  */
 export interface SignOptions extends CryptographyOptions {}
 
 /**
- * Options for the verify call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link verify}.
  */
 export interface VerifyOptions extends CryptographyOptions {}
 
 /**
- * Options for the wrapKey call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link wrapKey}.
  */
 export interface WrapKeyOptions extends CryptographyOptions {}
 
 /**
- * Options for the unwrap call to the CryptographyClient
+ * @interface
+ * An interface representing the optional parameters that can be passed to {@link unwrapKey}.
  */
 export interface UnwrapKeyOptions extends CryptographyOptions {}
 
 /**
- * Result of a decrypt operation
+ * @interface
+ * Result of the {@link decrypt} operation.
  */
 export interface DecryptResult {
   /**
-   * Result of the operation
+   * Result of the {@link decrypt} operation in bytes.
    */
   result: Uint8Array;
   /**
-   * Id of the key
+   * The ID of the KeyVault Key used to decrypt.
    */
   keyID?: string;
   /**
-   * Algorithm used
+   * The {@link EncryptionAlgorithm} used to decrypt.
```

Comment:
Please make similar updates to the description of `algorithm` in the below interfaces as well
- EncyrptResult
- SignResult
- VerifyResult
- WrapResult
- UnwrapResult